### ISSUE TITLE
stream() broken

### ISSUE THREAD

**Issue by lazics** [balderdashy/sails-mysql#247]
Hello,

the stream() function seems to be broken, the following simple example ( sails: 0.11.0 , sails-mysql: 0.11.0 ):

```
var stream = sails.models.item.stream( { } );
stream.on('data', function( data ) {
        console.log(data);
} );
stream.on('err', function( err ) {
        throw err;
});
```

fails with:

```
test1/node_modules/sails-mysql/node_modules/mysql/lib/protocol/Parser.js:82
        throw err;
              ^
TypeError: undefined is not a function
  at __STREAM__ (test1/node_modules/sails-mysql/lib/adapter.js:907:25)
  at afterwards (test1/node_modules/sails-mysql/lib/connections/spawn.js:84:5)
  at test1/node_modules/sails-mysql/lib/connections/spawn.js:40:7
  at Ping.onOperationComplete [as _callback] (test1/node_modules/sails-mysql/node_modules/mysql/lib/Pool.js:99:5)
  at Ping.Sequence.end (test1/node_modules/sails-mysql/node_modules/mysql/lib/protocol/sequences/Sequence.js:96:24)
  at Ping.Sequence.OkPacket (test1/node_modules/sails-mysql/node_modules/mysql/lib/protocol/sequences/Sequence.js:105:8)
  at Protocol._parsePacket (test1/node_modules/sails-mysql/node_modules/mysql/lib/protocol/Protocol.js:274:23)
  at Parser.write (test1/node_modules/sails-mysql/node_modules/mysql/lib/protocol/Parser.js:77:12)
  at Protocol.write (test1/node_modules/sails-mysql/node_modules/mysql/lib/protocol/Protocol.js:39:16)
  at Socket.<anonymous> (test1/node_modules/sails-mysql/node_modules/mysql/lib/Connection.js:96:28)
  at Socket.emit (events.js:107:17)
  at readableAddChunk (_stream_readable.js:163:16)
  at Socket.Readable.push (_stream_readable.js:126:10)
  at TCP.onread (net.js:538:20)
```

Line 907 of adapter.js is trying to call `sql.selectQuery`, which doesn't exist anywhere.

**Comment by sailsbot** [comment]
Thanks for posting, @lazics.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by sailsbot** [comment]
Thanks for posting, @lazics.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by devinivy** [comment]
This is an active bug.  https://github.com/balderdashy/sails-mysql/blob/master/lib/adapter.js#L907 references a method on `sql` that truly does not exist.

**Comment by particlebanana** [comment]
Fixed in #264
